src/components/Svg/TextPath.tsx

```diff
@@ -12,9 +12,10 @@ export default class TextPath extends React.Component<Props> {
   render() {
     if (!this.props.href || !this.props.href.match(idExpReg)) {
       // eslint-disable-next-line no-console
-      console.warn(
-        `Invalid \`href\` prop for \`TextPath\` element, expected a href like \`"#id"\`, but got: "${this.props.href}"`,
-      );
+      if (process.env.NODE_ENV !== 'production')
```

Comment:
In general, especially if you're running this as part of a web service, you don't want your libraries spamming the log, especially if you have a structured logging system (and you're not fully in control of the inputs because you're processing user data). 

A lot of node libraries use something akin to [debug](https://www.npmjs.com/package/debug) for this purpose, but I didn't want to go too far with this. I'm open to a different approach here if you have other ideas.